docs/moderne-cli/README.md

```diff
@@ -12,30 +15,113 @@ Boot 2 to Spring Boot 3.
 
 ### Download and configure the Moderne CLI
 
-1. Go to the [Moderne platform](https://app.moderne.io) and sign in. If you don't have an account, you can sign up for
-   free.
+1. Go to the [Moderne platform](https://app.moderne.io) and sign in. If you don't have an account, you can sign up for free.
+
+2. Download the CLI for your operating system:
+   <Tabs
+   defaultValue="mac"
+   values={[
+   {label: 'Mac', value: 'mac'},
+   {label: 'Linux', value: 'linux'},
+   {label: 'Windows', value: 'windows'},
+   ]}>
+   
+   <TabItem value="mac">
+
+   [moderne-cli-0.3.0.zip](https://drive.google.com/TODO)
+
+   </TabItem>
+   <TabItem value="linux">
+
+   [moderne-cli-0.3.0.zip](https://drive.google.com/TODO)
+
+   </TabItem>
+   <TabItem value="windows">
+
+   [moderne-cli-0.3.0.zip](https://drive.google.com/TODO)
+
+   </TabItem>
+   </Tabs>
+
+3. Once you have it downloaded, save it somewhere that your terminal can access. If you want it available in each of your terminal windows, consider updating your `PATH` to point to this location or aliasing `mod` to the location of the CLI. You could also save the file to a directory that's already on your `PATH` such as a `/usr/bin` directory.
+
+4. Ensure you can run the CLI by typing `mod`.
+
+  <details>
+  <summary>If everything is set up correctly, you should see output similar to the following:</summary>
+
+  ```shell
+  ➜  moderne-cli git:(main) mod
+    ▛▀▀▚▖  ▗▄▟▜
+    ▌   ▜▄▟▀  ▐
+    ▛▀▀█▀▛▀▀▀▀▜
+    ▌▟▀  ▛▀▀▀▀▜
+    ▀▀▀▀▀▀▀▀▀▀▀
+    Moderne CLI 1.0.0
```

Comment:
I manually put 1.0.0 in each of the text things as I expect they won't see snapshot and local dev machine text here.

Reply:
Perhaps better to add the CLI version to this script: https://github.com/moderneinc/springboot-migration-workshop/blob/main/src/scripts/versions.sh#L24